specs/CameraCaptureUI/CameraCaptureUI.md

```diff
@@ -0,0 +1,315 @@
+
+
+
+# 3. Description
+
+Winappsdk version of CameraCaptureUI is essentially the same as the existing public [CameraCaptureUI](https://learn.microsoft.com/en-us/uwp/api/windows.media.capture.cameracaptureui?view=winrt-22621) WinRT API, with a minor adjustment to the CameraCapture constructor to accept a windowID in the constructor for enhanced integration capabilities with desktop applications.
+
+The ***API*** includes the following key components:
+
+
+## 3.1 CameraCaptureUI
+
+Provides a full window UI for capturing audio, video, and photos from a camera. As well as controls for trimming video, time delayed capture, and camera settings.
+
+This example shows how to use the CameraCaptureUI Class to take a picture. The dialog with the user occurs when you call the asynchronous method CaptureFileAsync.
+```c#
+// Using Microsoft.Windows.Media.Capture.CameraCaptureUI API to capture a photo
+CameraCaptureUI dialog = new CameraCaptureUI(windowId);
+Size aspectRatio = new Size(16, 9);
+dialog.PhotoSettings.CroppedAspectRatio = aspectRatio;
+
+StorageFile file = await dialog.CaptureFileAsync(CameraCaptureUIMode.Photo);
+```
+Screenshot of the UI:
+
+
+![Screenshot](CameraCaptureUI_Snapshot.png)
+
+
+### 3.1.1 Constructor (The Change Introduced)
+| Name | Description |
+|-|-|
+| CameraCaptureUI(windowID) | Create a new CameraCaptureUI object.|
+```
+Note: After you create a new CameraCaptureUI object, you may want to set the PhotoSettings or VideoSettings property of the object before calling CaptureFileAsync.
+```
+
+### 3.1.2 Properties
+| Name | Description | Value |
+|-|-|-|
+|PhotoSettings| Provides settings for capturing photos. The settings include aspect ratio, image size, format, resolution, and whether or not cropping is allowed by the user interface (UI).|CameraCaptureUIPhotoCaptureSettings (An object containing settings for capturing photos) |
+|VideoSettings| Provides settings for capturing videos. The settings include format, maximum resolution, maximum duration, and whether or not to allow trimming.| CameraCaptureUIVideoCaptureSettings (An object that provides settings for capturing videos) |
+
+### 3.1.3 Methods
+
+| Name | Description | Parameters | Returns |
+|-|-|-|-|
+| CaptureFileAsync(CameraCaptureUIMode) | Launches the CameraCaptureUI user interface.| __mode__ CameraCaptureUIMode | __IAsyncOperation< StorageFile >__ |
+
+### 3.1.4 Remarks
+CameraCaptureUI provides a full window UI experience for capturing audio, video, and images. It provides controls for setting a time delay on photo captures, trimming video, and for adjusting the camera's settings such as video resolution, the audio device, brightness, and contrast.
+
+Call CaptureFileAsync to launch the UI. The user has control over when to start the capture. When the asynchronous CaptureFileAsync operation completes, a StorageFile object is returned. 
+
+`` Note: 
```

Comment:
Is this still true? Or is this a leftover UWP-ism?

Reply:
The camera app is currently a UWP application, and there's a team working on migrating it to the Windows App SDK. (not sure about the timelines). From what I understand, even in the Windows App SDK version of the API, the camera app functions as a full-trust application with built-in access to the camera. It seems unnecessary to explicitly specify this access in the manifest.

As for the note about potential issues with the Windows Application Certification Kit related to specifying access, we'll need to test this to see if it's still **relevant today.**

Also, since the camera app migration is a work in progress at the moment, maybe we should hold off on adding specific notes about this issue for now? Thoughts? @AjitSurana